packages/dds/matrix/src/test/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     },
     "include": [
         "./**/*"
-    ],
+, "memory/.mocharc.js"    ],
```

Comment:
Let's fix the indentation here. And I think the linter will ask we put a comma after that second entry in the array, and push the `],` down one line.